deployments/helm/hephaestus/values.yaml

```diff
@@ -114,88 +59,140 @@ controller:
     # Namespace labels where prometheus is running if different from app's namespace
     namespaceLabels: {}
 
-  # Configure pod security context
+  # Configure controller pod security context
   podSecurityContext: {}
 
-  # Configure container security context
-  containerSecurityContext: {}
+  # Annotations for controller pods
+  podAnnotations: {}
 
-  # Extra labels for manager pods
+  # Extra labels for controller pods
   podLabels: {}
 
-  # Annotations for manager pods
-  podAnnotations: {}
-
-  # Tolerations for manager pod assignment
+  # Tolerations for controller pod assignment
   tolerations: []
 
-  # Affinities for manager pod assignment
+  # Affinities for controller pod assignment
   affinity: {}
 
-  # Node labels for manager pod assignment
+  # Node labels for controller pod assignment
   nodeSelector: {}
 
-  # Manager pods priorityClassName
+  # Controller pods priorityClassName
   priorityClassName: ""
 
-  # Additional volume mounts
-  extraVolumeMounts: []
-
   # Additional volumes
   extraVolumes: []
 
-  # Extra environment variables provided to the controller container
-  extraEnvVars: []
-
-  # Number of manager instances to run. Leader election will be enabled
-  # whenever this value is greater than 1; this ensures that only one instance
-  # is active at a time.
-  replicaCount: 1
-
-  # Service account configuration
-  serviceAccount:
-    # Specifies whether a service account should be created
-    create: true
-    # Annotations to add to the service account
-    annotations: {}
-    # The name of the service account to use.
-    # If not set and create is true, a name is generated using the fullname template
-    name: ""
-
-# Log processor configuration
-logProcessor:
-  # Log processor image
-  image:
-    registry: ""
-    repository: timberio/vector
-    tag: 0.21.X-distroless-static
-    pullPolicy: IfNotPresent
-    pullSecrets: []
-
-  # Enable the deployment of the log processing sidecar
-  enabled: true
+  # Additional volume mounts added to both the manager and vector containers
+  extraVolumeMounts: []
```

Comment:
Feels a little "safer" to have this split into manager and vector extraVolumeMounts, but I also buy YAGNI

Reply:
We really only need this for right now but yeah I thought about that, and I think the middle ground is probably to have both a _unified_ and _separate_ `extraVolumeMounts`.